store/Cargo.toml

```diff
@@ -37,4 +37,5 @@ strum_macros = "0"
 
 [dev-dependencies]
 base64 = "0"
+tempdir = "0.3.7"
```

Comment:
we usually don't nail down this specifically.. but to the major version in this case so.. change to "0"

Reply:
Address in 598a7d7